### DISCUSSION TITLE
Would anyone like to add Carbon to GitHub Linguist?

### DISCUSSION THREAD

**Discussion by alecgargett** [carbon-language/carbon-lang#3370] [General]
Carbon appears to qualify to be added GitHub Linguist under the new criteria listed here:

https://github.com/github-linguist/linguist/issues/5756

as it has reached well over 2K here:
https://github.com/search?q=path%3A*.carbon&type=Code&ref=advsearch&l=&l=

The instructions on how to do so are here:
https://github.com/github-linguist/linguist/blob/master/CONTRIBUTING.md#adding-a-language

Once it has been added it will automatically appear in other projects that use GitHub Linguist such as Languish:
https://tjpalmer.github.io/languish/

I'd love to be able to track Carbon's growth over time using Languish and it'd be nice for others to see Carbon listed there too.

I'm not quite qualified to add Carbon myself so I'd greatly appreciate if someone else with the knowhow were to do it.

**Comment by chandlerc** [comment]
Seems reasonable to me to at least send a PR and see what the maintainers think. Feel free to file an issue? Or I can transfer this to an issue?

**Reply by alecgargett** [reply]
That'd be good if you can transfer it for me.
